src/main/java/lotto/domain/WinningStatus.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class WinningStatus {
+    private static final long DEFAULT_WINNING_COUNT = 0L;
+    private final Map<Rank, Long> value;
+
+    public WinningStatus(List<Rank> ranks) {
+        this.value = ranks.stream()
+                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
+    }
```

Comment:
정적 팩터리 메서드를 사용한다면 생성자의 접근 제어자를 private 으로 변경해
정적 팩터리 메서드를 사용하도록 하면 어떨까요? 😄 
```suggestion
    private WinningStatus(List<Rank> ranks) {
        this.value = ranks.stream()
                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
    }
```

Reply:
맞습니다. 정적 팩토리 메서드를 사용하여 생성하니 생성자를 통해 직접 생성하는것은 불필요하겠네요